docker-compose.yml

```diff
@@ -2,11 +2,39 @@ version: '3'
 
 services:
   db:
-    image: postgres:12
+    container_name: db
+    image: postgres:13
     environment:
       POSTGRES_DB: northwind
       POSTGRES_USER: northwind_user
       POSTGRES_PASSWORD: thewindisblowing
     volumes:
-      - ./dbdata:/var/lib/postgresql/data
+      - /var/lib/postgresql/data:/var/lib/postgresql/data
```

Comment:
It will be better to leave the volume location as local subfolder as before:

````
volumes:
      - ./dbdata:/var/lib/postgresql/data
````

i.e, the folder is kept locally. Otherwise, bad things may happen:
* the docker container my overwrite your own main postgres database folder
* if the `/var/lib/postgresql/data` folder does not exist (for example, on a Mac; such a my computer), it will also fail

Reply:
Hum, you marked this as solved, but did  not modify anything. Is this a mistake, or do you have an explanation for this?

Reply:
I'm sorry, made changes.